tests/foreman/api/test_parameters.py

```diff
@@ -14,11 +14,51 @@
 import pytest
 
 
+@pytest.fixture
+def param_name():
```

Comment:
Request to add description in the fixture.

Reply:
Generate name for common parameter and return to caller function.
```suggestion
def param_name():
    """Generate name string for common parameter and return to caller"""
```